contracts/src/v0.8/dev/vrf/testhelpers/VRFCoordinatorV2PlusUpgradedVersion.sol

```diff
@@ -633,44 +633,12 @@ contract VRFCoordinatorV2PlusUpgradedVersion is
     emit CoordinatorRegistered(target);
   }
 
-  function migrate(uint256 subId, address newCoordinator) external nonReentrant {
-    if (!isTargetRegistered(newCoordinator)) {
-      revert CoordinatorNotRegistered(newCoordinator);
-    }
-    (uint96 balance, uint96 ethBalance, , address owner, address[] memory consumers) = getSubscription(subId);
-    require(owner == msg.sender, "Not subscription owner");
-    require(!pendingRequestExists(subId), "Pending request exists");
-
-    V1MigrationData memory migrationData = V1MigrationData({
-      fromVersion: migrationVersion(),
-      subId: subId,
-      subOwner: owner,
-      consumers: consumers,
-      linkBalance: balance,
-      ethBalance: ethBalance
-    });
-    bytes memory encodedData = abi.encode(migrationData);
-    deleteSubscription(subId);
-    IVRFCoordinatorV2PlusMigration(newCoordinator).onMigration{value: ethBalance}(encodedData);
-
-    // Only transfer LINK if the token is active and there is a balance.
-    if (address(LINK) != address(0) && balance != 0) {
-      require(LINK.transfer(address(newCoordinator), balance), "insufficient funds");
-    }
-
-    // despite the fact that we follow best practices this is still probably safest
-    // to prevent any re-entrancy possibilities.
-    s_config.reentrancyLock = true;
-    for (uint256 i = 0; i < consumers.length; i++) {
-      IVRFMigratableConsumerV2Plus(consumers[i]).setCoordinator(newCoordinator);
-    }
-    s_config.reentrancyLock = false;
-
-    emit MigrationCompleted(newCoordinator, subId);
+  function migrate(uint256 /* subId */, address /* newCoordinator */) external view nonReentrant {
+    require(false, "not implemented");
   }
```

Comment:
@iljapavlovs removed this logic because my recent change above brought contract size over the limit again. Do we need `migrate` on this upgraded version?